content/en/docs/concepts/workloads/controllers/cron-jobs.md

```diff
@@ -162,17 +162,15 @@ For another way to clean up jobs automatically, see [Clean up finished jobs auto
 
 ### Time zones
 
-For CronJobs with no time zone specified, the {{< glossary_tooltip term_id="kube-controller-manager" text="kube-controller-manager" >}} interprets schedules relative to its local time zone.
+For CronJobs with no time zone specified, the {{< glossary_tooltip term_id="kube-controller-manager" text="kube-controller-manager" >}}
+interprets schedules relative to its local time zone.
 
-{{< feature-state for_k8s_version="v1.25" state="beta" >}}
+{{< feature-state for_k8s_version="v1.27" state="stable" >}}
```

Comment:
For stable, move the feature state shortcode just under the heading, and put the detail about CronJobs with no time zone specified into a note callout later on in the **Time zones** section.

Consider moving the caution callout about `TZ` / `CRON_TZ` to be the first item in **CronJob limitations**, and if you want you can hyperlink to that from somewhere within the Time zones section. We're really framing that now as a shortcoming rather than anything close to a feature.

Reply:
the feature state should be under the Time zones section and not under the CronJob main heading, right?